src/index.js

```diff
@@ -0,0 +1,219 @@
+const state = {
+    tempCount : "",
+    city: "Seattle",
+    lat: "",
+    lon: "", 
+    
+}
+
+const landscapeImg = { 
+    "1" : "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+    "2" : "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+    "3" : "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+    "4" : "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+    "5" : "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+};
+
+const skyImg = { 
+    "0" : "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+    "1": "☁️ ☁️ ☁️ ☀️ ☁️ ☁️",
+    "2" : "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+    "3" : "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"
+};
```

Comment:
Numbers as keys work fine but I think it would be as easy and more readable to call them `"sunny"`, `"cloudy"`, etc-- 😉